activesupport/lib/active_support/cache/redis_cache_store.rb

```diff
@@ -451,21 +452,18 @@ def truncate_key(key)
           end
         end
 
-        def deserialize_entry(serialized_entry, raw:)
-          if serialized_entry
-            if raw
-              Entry.new(serialized_entry)
-            else
-              Marshal.load(serialized_entry)
-            end
+        def deserialize_entry(payload, raw:)
+          if payload && raw
+            return Entry.new(payload)
           end
+          super(payload)
```

Comment:
Switching to an explicit return here seems cosmetic.

```suggestion
            Entry.new(payload)
          else
            super(payload)
          end
```